website/source/api/secret/databases/index.html.md

```diff
@@ -397,3 +397,225 @@ $ curl \
   }
 }
 ```
+
+## Create Static Role
```

Comment:
Should mention that not all database plugins support this.

Reply:
Added in 2ca4849e3